src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs

```diff
@@ -265,55 +270,63 @@ private static bool IsRequestInstrumented(HttpWebRequest request)
 
     private static void ProcessRequest(HttpWebRequest request)
     {
-        if (!WebRequestActivitySource.HasListeners() || !Options.EventFilterHttpWebRequest(request))
+        // No subscribers to the ActivitySource or User provider Filter is
+        // filtering this request.
+        var skipTracing = !WebRequestActivitySource.HasListeners() || !Options.EventFilterHttpWebRequest(request);
+
+        if (skipTracing && !HttpClientDuration.Enabled)
```

Comment:
```suggestion
        // There are subscribers to the ActivitySource and no user-provided filter is
        // filtering this request.
        var enableTracing = WebRequestActivitySource.HasListeners() && Options.EventFilterHttpWebRequest(request);

        if (!enableTracing && !HttpClientDuration.Enabled)
```
A bit more readable in my opinion, but maybe a matter of taste.